source/javascripts/app/_copy.js

```diff
@@ -0,0 +1,15 @@
+//= require ../lib/_clipboard.min
+
+$(document).ready(function(){
+	$('.highlight').prepend('<a href="javascript: void(0);" class="code-copy-button" ><img class="clippy" width="13" src="../images/clippy.svg" alt="Copy to clipboard"></button>');
```

Comment:
The html you're inserting here doesn't match - you start with an anchor tag but end with a button. Also, the img tag isn't closed. Lastly, it's generally a good idea to break long lines into multiple shorter lines. In this case, you can just concatenate the strings like:
```
('<a href="javascript: void(0);" class="code-copy-button" >' +
'    <img class="clippy" width="13" src="../images/clippy.svg" alt="Copy to clipboard">' +
'</button>');
```
This would also help you detect that the html tags don't match.